dockerfile/cuda11.1.1.dockerfile

```diff
@@ -38,6 +39,7 @@ RUN apt-get update && \
     openssh-client \
     openssh-server \
     pciutils \
+    sudo \
```

Comment:
just curious, why we need sudo inside docker

Reply:
From Liwei's requirement

Reply:
`iproute2` is depended by `ibdev2netdev`
as for `sudo`, some environments run containers as a normal user (azure pipeline, some ai platforms) which has privileged permission but cannot perform privileged operations without `sudo`